Core/src/IfcGeometryConverter/ConverterBuw.h

```diff
@@ -340,6 +340,10 @@ OIP_NAMESPACE_OPENINFRAPLATFORM_CORE_IFCGEOMETRYCONVERTER_BEGIN
 										geometry->meshDescription.vertices, geometry->meshDescription.indices);
 								}
 
+								// append meshGridLines to polylines (temporal, both are handled similar in further treatment)
+								// ToDo: meshGridLines should be handled separately in order to enable / disable their visibility in the viewport
+								std::copy(itemData->meshGridLines.begin(), itemData->meshGridLines.end(), std::back_inserter(itemData->polylines));
+

```

Comment:
I wouldn't copy the data into polylines, but rather call the `ConverterBuwT::insertPolylineIntoBuffers` function, similarly to the lines for `itemData->polylines` - therefore only copying into buffers.

Reply:
Done in commit c9eb7cb.